### DISCUSSION TITLE
Can I read link href attribute?

### DISCUSSION THREAD

**Discussion by Jakooth** [nvaccess/nvda#15782] [Q&A] [answered]
Is there a ways with NVDA keyboard shortcuts to read a particular link href attributes so I know where it navigates to?

The context is I'm not a screen reader user, but I'm an accessibility reviewer and I use NVDA when testing.
I give instructions in my reports on how to reach a particular page from a link without label, for example, I have a list of products with pattern /products/details/product-id, where last part of the link is not reliable, because a product might not exits in another person testing environment.
If I want my reports to be inclusive, I can give instruction, like, find and activate any link with URL leading to /products/details/, so let say a person with vison loss can perceive the report and navigate to the problematic page the same way I do.

In addition I'm passing annual security trainings and one of the major topics is identifying phishing URLs, where one perceive a text, like, Open user account menu, but the URL behind it could be malicious, because the domain is not what one can expect.
I'm wondering how someone who is blind can be safe in this scenario?

**Comment by hwf1324** [comment, marked as answer]
The URL of the link where the current navigation object is located can be read out using NVDA+k, see #14583 for details.

**Reply by Jakooth** [reply]
Great! Thanks for the quick reply!
